### ISSUE TITLE
A11y_Screenreader reading radio button count incorrectly.

### ISSUE THREAD

**Issue by SachinChavhan317** [angular/components#23793]
GitHubTags: #A11yMAS; #A11yTCS; #A11ySev2; #DesktopWeb; #WCAG1.3.1; #ChromiumEdge; #Narrator; #Facility Master; #GH-Facility Master-Web-July2021; #Win10; 

**Environment details:**
Application Name: Facility Master
Windows version: Windows 10
Chromium Edge :Version 90.0.818.46 (Official build) (64-bit)

**Repro Steps:**
1. Launch the application 
2.  Tab navigate to Launch app & hit enter.
3.  Tab navigate to Colocations tab & hit enter.
4.  Tab navigate to table & select any entry.
5.  Tab navigate to Edit & hit enter.
6.  Verify that Screenreader is reading radio button count correctly or not.

**Actual:**
Screen reader reading radio button count incorrectly.

NOTE : This issue is reproducing through out the application where ever the radio buttons are present.

No issue is observed with NVDA

**Expected:**
Screen reader should announce radio button count correctly

**User Impact:**
Screen reader user will difficult to understand radio button count.
